### PR TITLE
Using @nobles/hashes

### PR DESCRIPTION
**Pull request by AlexAndrei98**

### CODE REVIEW

File: src/ready.ts [hunk 1]
```diff
@@ -2,10 +2,6 @@ import _sodium from 'libsodium-wrappers-sumo';
 
 export const ready: () => Promise<void> = async () => {
     try {
```
Comment (lenkan) [inline src/ready.ts:4]: We do not need the try catch anymore since the catch block now just awaits _sodium.ready again.

Comment (AlexAndrei98) [reply src/ready.ts:4]: You are right! will refactor this!

File: package-lock.json
[the remark this answers is not in the document]
Comment (AlexAndrei98) [reply package-lock.json:1]: Good catch! Will try to re-install!

**Comment by lenkan** [comment]
@AlexAndrei98 I think this just needs a rebase to verify the integration tests are working.

**Comment by AlexAndrei98** [comment]
> @AlexAndrei98 I think this just needs a rebase to verify the integration tests are working.

@lenkan I tried to rebase from development 🤞🏽

**Comment by pfeairheller** [comment]
The commit history is a mess, could you please squash this so we can review it properly?

**Comment by AlexAndrei98** [comment]
closing in favor of #174

**Pull request closed without merging**